### PR TITLE
server.py and pid.py changes. disabling tl_detector. setting Pi to zero for both controllers

### PR DESCRIPTION
**Pull request by asimonov**
…ontrol to be send to simulator

### CODE REVIEW

**Review by vguerra — APPROVED** [review]
lgtm. 

I have applied this very same patch before locally.

**Comment by asimonov** [comment]
this configuration works for me, i.e. the car can drive around many laps on my macbook pro (4 cores). 
however, if I run visualisation then car comes to a stop by the end of the track. but starts moving again if I kill visualisation.

**Review by yhoazk — APPROVED** [review]
Tested the code and I got similar results.

File: ros/src/twist_controller/pid.py [hunk 1]
```diff
@@ -20,18 +20,16 @@ def reset(self):
...
         derivative = (error - self.last_error) / sample_time
 
         y = self.kp * error + self.ki * self.int_val + self.kd * derivative
         val = max(self.min, min(y, self.max))
 
+        self.last_error = error
         if val > self.max:
             val = self.max
         elif val < self.min:
             val = self.min
```
Comment (vguerra) [inline ros/src/twist_controller/pid.py]: I think l.30 to l.33 are useless.. l.27 already makes sure that val is within [self.min, self.max] range.

So we can safely remove this.

Comment (asimonov) [reply ros/src/twist_controller/pid.py]: good point
just added another commit

**Pull request merged**